.dvc/config

```diff
@@ -2,3 +2,5 @@
     remote = storage
 ['remote "storage"']
     url = https://remote.dvc.org/dataset-registry
+['remote "local"']
```

Comment:
please remove this

Reply:
Removed.
Included also line 6 url
Remote test in local